### PR TITLE
api-database

### PR DESCRIPTION
**Pull request by codeWhizperer**

### CODE REVIEW

File: api/config/pool.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+         console.log(response);
+         pool.end();
+     })
+     .catch((error)=>{
+         console.log(error)
+         pool.end();
+     })
+ }
+ 
+ const dropUserTable = () =>{
```
Comment (TheoOkafor) [inline api/config/pool.js:66]: You have two of these functions `dropUserTable`. The one above and this one

File: api/config/query.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ 
+ module.exports = {
+   /**
+    * DB Query
+    * @param {object} req
+    * @param {object} res
+    * @returns {object} object
+    */
+   query(quertText, params) {
+     return new Promise((resolve, reject) => {
```
Comment (TheoOkafor) [inline api/config/query.js:11]: Why was it necessary to wrap the response of this function in a promise?

File: api/controller/auth.js [hunk 3]
```diff
@@ -87,4 +49,50 @@ return response.status(200).json({message:'User Successfully Logged In', info:to
 
 
 
-module.exports = {signup, login}
\ No newline at end of file
+
+
+
+
```
Comment (TheoOkafor) [inline api/controller/auth.js:55]: Check your linting tools. I am having a challenge reading this file because it's not linted.

**Review by TheoOkafor — COMMENTED** [review]
Good progress. Well done.

Check your linting too. have these files linted and respond to my other comments

File: api/config/pool.js (hunk 1 above)
Comment (codeWhizperer) [reply api/config/pool.js:66]: I will correct that

File: api/config/query.js (hunk 2 above)
Comment (codeWhizperer) [reply api/config/query.js:11]: The promise helps to query the database, the method returns a promise in the controller... I got it from the tutorial I read online but the end-points works just fine without the query.js

**Pull request merged**